examples/cms-takeshape/README.md

```diff
@@ -0,0 +1,156 @@
+
+Next, add these fields (you don't have to modify the settings unless specified):
+
+- **Single Line** widget: Set the title as **Title**.
+- **Markdown** widget: Set the title as **Content**.
+- **Single Line** widget: Set the title as **Excerpt**.
+- **Asset** widget: Set the title as **Cover Image**.
+- **Date** widget: Set the title as **Date**.
+- **Single Line** widget: Set the title as **Slug**.
+- **Relationship** widget: Set the title as **Author**, then set **Relationship Type** to `Single` and check the `Author` checkbox under **Allowed Content Types**.
+
+When you’re done, click "Create Content Type".
+
+### Step 4. Populate Content
+
+Select **Author** and create a new record.
+
+- You just need **1 Author record**.
+- Use dummy data for the text.
+- For the image, you can download one from [Unsplash](https://unsplash.com/).
+
+When you’re done, make sure to click **Enabled** under **Workflow Status**.
+
+Next, select **Post** and create a new record.
+
+- We recommend creating at least **2 Post records**.
+- Use dummy data for the text.
+- You can write markdown for the **Content** field.
+- For the images, you can download ones from [Unsplash](https://unsplash.com/).
+- Pick the **Author** you created earlier.
+
+When you’re done, make sure to click **Enabled** under **Workflow Status**.
+
+### Step 5. Set up environment variables
+
+From the dropdown next to the project name, click **API Keys**.
+
+Create a new API Key with the **Read** permission.
+
+Next, copy the `.env.example` file in this directory to `.env` (which will be ignored by Git):
+
+```bash
+cp .env.example .env
+```
+
+Then set each variable on `.env`:
+
+- `NEXT_EXAMPLE_CMS_TAKESHAPE_API_KEY` should be the API token you just copied.
+- `NEXT_EXAMPLE_CMS_TAKESHAPE_PROJECT_ID` should be the project ID, which is a substring in the project page URL: `https://app.takeshape.io/projects/<project-id>/...`
+- `NEXT_EXAMPLE_CMS_TAKESHAPE_PREVIEW_SECRET` can be any random string (but avoid spaces), like `MY_SECRET` - this is used for [the Preview Mode](/docs/advanced-features/preview-mode.md).
+
+Your `.env` file should look like this:
+
+```bash
+NEXT_EXAMPLE_CMS_TAKESHAPE_PROJECT_ID=...
+NEXT_EXAMPLE_CMS_TAKESHAPE_API_KEY=...
+NEXT_EXAMPLE_CMS_TAKESHAPE_PREVIEW_SECRET=...
+```
+
+### Step 6. Run Next.js in development mode
+
+```bash
+npm install
+npm run dev
+
+# or
+
+yarn install
+yarn dev
+```
+
+Your blog should be up and running on [http://localhost:3000](http://localhost:3000)! If it doesn't work, post on [GitHub discussions](https://github.com/zeit/next.js/discussions).
+
+### Step 7. Try preview mode
+
+On TakeShape, create a new post like before. But **DO NOT** click **Enabled** under **Workflow Status**.
+
+Now, if you go to `http://localhost:3000/posts/<slug>` (replace `<slug>`), you won’t see the post. However, if you use the **Preview Mode**, you'll be able to see the change ([Documentation](/docs/advanced-features/preview-mode.md)).
```

Comment:
You can still see the blog post because the API is returning it for some reason. It seems that in TakeShape's GraphQL API, if you have `filter` then `onlyEnabled` is ignored. Looking into this issue.

Reply:
It was fixed by:

```
{
  filter: { term: { name: "foo" } },
  where: { _status: { eq: "enabled" } }
}
```